domains/kikemendez.json

```
@@ -0,0 +1,11 @@
+{
+    "owner": {
+      "username": "Kikeme98",
+      "email": "l.e.mendez1998@gmail.com"
+    },
+    "description": "My personal site portfolio",
+    "repo": "https://github.com/Kikeme98/react-frontend-dev-portfolio.git",
+    "record": {
+      "CNAME": "kikeme98.github.io/react-frontend-dev-portfolio"

```

CNAME can't contain `/`
```suggestion
      "CNAME": "kikeme98.github.io"
```